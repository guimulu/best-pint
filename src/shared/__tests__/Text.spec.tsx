import * as React from "react";
import { createSnapshot } from "jest-react-fela";

import { Text } from "..";

describe("Text component", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Text fontSize={12} />);
    expect(snapshot).toMatchSnapshot();
  });
});
