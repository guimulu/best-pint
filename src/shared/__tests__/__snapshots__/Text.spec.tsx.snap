// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component renders correctly 1`] = `
".a {
  font-size: 12px;
}
.b {
  color: #6c8ead;
}
.c {
  text-align: left;
}
.d {
  font-weight: normal;
}
.e {
  padding: 4px;
}
.f {
  margin-bottom: 0px;
}


<div className=a b c d e f />;
"
`;
